dynatrace-operator/values.yaml

```diff
@@ -14,31 +14,101 @@
 
 platform: "kubernetes"
 
+### base properties
+
 name: "dynakube"
 apiUrl: ""
-apiToken: ""
-paasToken: ""
```

Comment:
Don't remove fields in this file, this is a breaking change for customers who already installed the Operator.

Reply:
if we start to talk about breaking change, we can go to 1.x :trollface: